aioftp/client.py

```diff
@@ -120,7 +120,8 @@ def __init__(self,
                  read_speed_limit=None,
                  write_speed_limit=None,
                  path_timeout=None,
-                 path_io_factory=pathio.PathIO):
+                 path_io_factory=pathio.PathIO,
+                 list_compatibility_level=2):
```

Comment:
Actually, I left that as an option for easier testing, but I also made an automatic fallback.